canonical-web-and-design/ubuntu.com
# New advantage legacy user subscriptions

## Done

- Add legacy user subscriptions to the /advantage/user-subscriptions endpoint

## QA

- Have a legacy subscription and be logged in
- Go to: https://ubuntu-com-10282.demos.haus/user-subscriptions?test-backend=true
- You will see your user subscription, one for each legacy purchase you have

## Issue / Card

Fixes #https://github.com/canonical-web-and-design/commercial-squad/issues/179
